packages/dso-toolkit/src/components/advanced-select/advanced-select.models.ts

```diff
@@ -0,0 +1,44 @@
+export type AdvancedSelectVariant = "primary" | "success" | "info" | "warning" | "danger" | "error" | "attention";
```

Comment:
Waarom "danger" en "error" naast elkaar? Die resulteren beide in een rode bullet voor de option. Ik zie dat bij label dit ook zo is, mogelijk is dat legacy, maar omdat dit een nieuw component is, zou ik kiezen voor 1 van de 2. Misschien weet @hansgrimm waarom danger en error nog naast elkaar bestaan en welke van de 2 de actuele varaint is. Is 1 van de 2 deprecated?

Reply:
Ik vond dit zelf ook niet duidelijk. In meerdere componenten worden verschillende varianten gedefineerd dus heb ik de uitgebreide variant gepakt en overgenomen. Nu ben ik wel benieuwd welke dan nu de juiste is. @hansgrimm Heb jij een idee?

Is het wellicht een goed idee om op 1 plek de juiste type te defineren en deze overal te gebruiken? Dan heeft elk component de zelfde waardes voor varianten.

Reply:
Ja dat is nog een dingetje, meer semantisch dan functioneel. Danger en error zijn verschillende dingen qua betekenis, en ze zijn toevallig allebei rood. Ik vind het wel een goede reden om hier weer eens over te bomen, want het komt redundant over. goed punt dus.

Reply:
Ik heb dit overgenomen van Badge. Ideaal is het zeker niet. Laten we hier eens met de UXers over praten. En ook over variant vs. status...